gstools/field/summator.pyx

```diff
@@ -12,31 +12,26 @@ from libc.math cimport sin, cos
 cimport numpy as np
 
 
-DTYPE = np.double
-ctypedef np.double_t DTYPE_t
-
-
 def summate(
-    const double[:,:] cov_samples,
+    const double[:, :] cov_samples,
     const double[:] z_1,
     const double[:] z_2,
-    const double[:,:] pos
+    const double[:, :] pos
     ):
-    cdef int i, j, d, X_len, N
+    cdef int i, j, d
     cdef double phase
-    cdef int dim
-    dim = pos.shape[0]
+    cdef int dim = pos.shape[0]
 
-    X_len = pos.shape[1]
-    N = cov_samples.shape[1]
+    cdef int X_len = pos.shape[1]
+    cdef int N = cov_samples.shape[1]
 
-    cdef double[:] summed_modes = np.zeros(X_len, dtype=DTYPE)
```

Comment:
So is using `dtype` really redundant here? It seems I know to little about cython-numpy integration.